### PR TITLE
increase uv threadpool size

### PR DESCRIPTION
**Pull request by ExtraHash**

### CODE REVIEW

**Comment by zpalmtree** [comment]
Can you confirm that this fixed the timeout issues for you?

**Comment by ExtraHash** [comment]
I no longer experience any timeouts with this if I'm on a decently fast connection. I'm still experiencing occasional timeouts if I'm on a very slow (~4g speeds) connection but I think that's a separate issue.

File: lib/index.ts [hunk 1]
```diff
@@ -17,3 +17,7 @@ export { validateAddresses, validatePaymentID } from './ValidateParameters';
 export { TransactionInput, DaemonType, DaemonConnection } from './Types';
 export { MixinLimit, MixinLimits } from './MixinLimits';
 export { Config } from './Config';
+
+// this is to keep pesky timeout errors away
+// see https://stackoverflow.com/questions/24320578/node-js-get-request-etimedout-esockettimedout/37946324#37946324
+process.env.UV_THREADPOOL_SIZE = '256';
```
Comment (zpalmtree) [inline lib/index.ts]: Can you clarify why you're using a string here? All the examples I seem to see on the internet are using a number.

**Comment by ExtraHash** [comment]
process.env takes a string by design, it's typed like this:

```{ [key: string]: string } ```

see

https://github.com/microsoft/TypeScript/issues/11239

and

https://github.com/microsoft/TypeScript/issues/11239

File: lib/index.ts (hunk 1 above)
Comment (ExtraHash) [reply lib/index.ts]: process.env takes a string by design, it's typed like this:

```{ [key: string]: string } ```

see

https://github.com/microsoft/TypeScript/issues/11239

and

https://github.com/microsoft/TypeScript/issues/11239

**Pull request merged**